app/src/modules/team/team.actions.js

```diff
@@ -4,6 +4,39 @@ import envConfig from '../../environment/base';
 import API from '../api';
 
 const teamResource = API('teams');
+const competitionsResource = API('competitions');
+
+export const COMPETITIONS_REQUEST = 'COMPETITIONS_REQUEST';
+export const COMPETITIONS_SUCCESS = 'COMPETITIONS_SUCCESS';
+export const COMPETITIONS_FAILURE = 'COMPETITIONS_FAILURE';
+
+export function getCompetitions() {
+  return {
+    [CALL_API]: competitionsResource.get('?season=2016', {
+      types: [COMPETITIONS_REQUEST, COMPETITIONS_SUCCESS, COMPETITIONS_FAILURE],
+      headers: {
+        'X-Auth-Token': envConfig.api.xAuthToken
+      },
```

Comment:
To trzeba przenieść w apiResource i dodawać do każdego zapytania, razem z credentials